plugin.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "gdpr",
-    "version": "0.1.6",
+    "version": "0.2.0",
```

Comment:
The plugin version has to be updated in the Makefile as well

Reply:
Fixed in [0fe9fca](https://github.com/dataiku/dss-plugin-gdpr/pull/1/commits/0fe9fcad0ad7b1168226091f0cb6a5e093991ca1)